test/integration/targets/win_scheduled_task/tasks/tests.yml

```diff
@@ -211,3 +211,124 @@
     that:
     - remove_scheduled_task_new_path_1.msg == 'Task does not exist'
   when: in_check_mode
+
+
+# Test scheduled task RunAs and RunLevel options
+
+- name: Remove potentially leftover run options task 1
+  win_scheduled_task: &wstr1_absent
```

Comment:
so what does the `&wstr1_absent` actually do, does it store some variables, add optionality or something?